integration/test_ml.py

```diff
@@ -0,0 +1,361 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Integration tests for firebase_admin.ml module."""
+import re
+import os
+import shutil
+import random
+import tempfile
+import pytest
+
+
+from firebase_admin import ml
+from firebase_admin import exceptions
+from tests import testutils
+
+
+# pylint: disable=import-error,no-name-in-module
+try:
+    import tensorflow as tf
+    _TF_ENABLED = True
+except ImportError:
+    _TF_ENABLED = False
+
+
+NAME_ONLY_ARGS = {
+    'display_name': 'TestModel123_{0}'.format(random.randint(1111, 9999))
```

Comment:
Perhaps define a helper:

```
import random
import string

def _random_identifier(prefix):
  suffix = ''.join([random.choice(string.ascii_letters + string.digits) for n in xrange(8)])
  return '{0}_{1}'.format(prefix, suffix)
```

PS: Just realized that this is only important if display name uniqueness is enforced by the ML API. If not constant identifiers are fine. Sorry, I forgot to ask earlier.

Reply:
Uniqueness is enforced. Done. But I changed to range because Python3 doesn't have xrange.